python/lsst/pipe/base/pipelineIR.py

```diff
@@ -664,18 +665,38 @@ def process_args(argument: Union[str, dict]) -> dict:
         else:
             self.imports = [ImportIR(**process_args(tmp_import))]
 
+        self.merge_pipelines([fragment.toPipelineIR() for fragment in self.imports])
+
+    def merge_pipelines(self, pipelines: Iterable[PipelineIR]) -> None:
+        """Merge one or more other `PipelineIR` objects into this object.
+
+        Parameters
+        ----------
+        pipelines : `Iterable` of `PipelineIR` objects
+            An `Iterable` that contains one or more `PipelineIR` objects to
+            merge into this object.
+
+        Raises
+        ------
+        ValueError : Raised if there is a conflict in instrument specifications
```

Comment:
Missing period at the end of the sentence.

Reply:
Isn't the syntax also wrong for the numpydoc? Doesn't the "Raised if" go on the next line with no colon.

Reply:
According to https://developer.lsst.io/python/numpydoc.html, yes, you are correct.